src/java.base/share/classes/java/lang/ref/Reference.java

```diff
@@ -388,11 +388,21 @@ boolean refersToImpl(T obj) {
     private native boolean refersTo0(Object o);
 
     /**
-     * Clears this reference object.  Invoking this method will not cause this
-     * object to be enqueued.
+     * Clears this reference object. Invoking this method does not enqueue this
+     * object, and the garbage collector will no longer clear or enqueue this
+     * object.
      *
-     * <p> This method is invoked only by Java code; when the garbage collector
-     * clears references it does so directly, without invoking this method.
+     * <p>When the garbage collector or the {@link #enqueue()} method clear
+     * references they do so directly, without invoking this method.
+     *
+     * @apiNote
+     * There is a potential race condition with the garbage collector. When this
+     * method is called, the garbage collector may already be in the process of
+     * (or already completed) clearing and/or enqueueing this reference.
+     *
```

Comment:
Either this is an extra blank line, or you need a <p> here.

Reply:
Removed; I thought it looked better for the API note to be a single paragraph.